packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -180,8 +180,15 @@ export default class MigrateVmsModalBody extends BaseComponent {
       return
     }
     const { pools, pifs } = this.props
-    const defaultMigrationNetworkId = find(pifs, pif => pif.$host === host.id && pif.management).$network
-    const defaultSrId = pools[host.$pool].default_SR
+    const pool = pools[host.$pool]
+    const migrationNetwork = pool.otherConfig['xo:migrationNetwork']
+    const defaultMigrationNetworkId = find(
```

Comment:
```js
const defaultMigrationNetworkId = defined(pool.otherConfig['xo:migrationNetwork'], find(pifs, pif => pif.$host === host.id && pif.management).$network)
```

Reply:
if `pool.otherConfig['xo:migrationNetwork']` is defined doesn't mean that the network can be the default migration network because it depends on the host's destination.

Reply:
Ah, I understand now, thank you :+1:  We need also to ensure that the destination host is also attached to the default network.

So, probably the usage of `find` isn't appropriate here because the management PIF will always be selected if it is positioned before the default network PIF in the `pifs` collection.